test/helper/setup.bash

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# -------------------------------------------------------------------
+
+function __initialize_variables() {
+  function __check_if_set() {
+    if [[ ${!1+set} != 'set' ]]
+    then
+      echo "ERROR: (helper/setup.sh) '${1:?No variable name given to __check_if_set}' is not set" >&2
+      exit 1
+    fi
+  }
+
+  __check_if_set 'REPOSITORY_ROOT'
+  __check_if_set 'IMAGE_NAME'
+  __check_if_set 'CONTAINER_NAME'
```

Comment:
Yes, these variables are required to be set for new tests to run. This enables tests to be sure that these are set, and as a consequence, tests can use them without worrying that they are not set. There is not much effort, since Make setes two of the three values. You will only need to provide `CONTAINER_NAME`, which I believe to be completely fine.

You're right about the dummy value earlier.